color_refinement_helper.py

```diff
@@ -138,6 +141,82 @@ def get_twins(g: Graph):  # -> List[(Vertex, Vertex)], List[(Vertex, Vertex)]:
     return twins, false_twins
 
 
+def graph_to_modules(graph: Graph) -> ModularDecomposition:
+    vertices = graph.vertices
+    vertices_in_any_module = []
+    modular_decomposition = []
+
+    for vertex in vertices:
+        if vertex in vertices_in_any_module:
+            continue
+
+        module = [vertex]
+        vertices_in_any_module.append(vertex)
+
+        neighbours = vertex.neighbours
+
+        other_vertices = [vertex for vertex in vertices if vertex not in vertices_in_any_module]
+        for other_vertex in other_vertices:
+            other_neighbours = set(other_vertex.neighbours) - {vertex}
```

Comment:
Kun je hier niet de `are_twins` methode gebruiken?

Reply:
Ik weet niet wat sneller is: `are_twins` of wat ik doe. Dit is echter geschreven ter vervanging van `are_twins` en `get_twins`, want tweelingen worden ook altijd gevonden door `get_modules`. Wordt `are_twins` / `get_twins` nog ergens in 'productie' gebruikt?